static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -325,7 +332,7 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
   }
 
   // No platform or no docs
-  if (!currentPlatform || !docs || !dsn) {
+  if (!currentPlatform || !docs || !dsn || !hasDocs) {
```

Comment:
what about the existing `docs` object? Could you add a comment explaining the difference

Reply:
updated